challenges/functions.js

```diff
@@ -6,17 +6,28 @@
   * The last parameter accepts a callback 
   * In the body of the function return the callback with the two parameters that you created
 */
-
+let  Func = (param1,param2,cb) => {
+  return cb(param1 + param2)
```

Comment:
* The function should return the callback with the two parameters that you created. Like so:
return `cb(param1, param2);`
* Following the instructions is very important in this assignment.